gcloud/pubsub/_gax.py

```diff
@@ -159,14 +165,18 @@ def topic_publish(self, topic_path, messages):
         message_pbs = [_message_pb_from_dict(message)
                        for message in messages]
         try:
-            response = self._gax_api.publish(topic_path, message_pbs)
+            event = self._gax_api.publish(topic_path, message_pbs)
+            if not event.is_set():
+                import pdb; pdb.set_trace()
```

Comment:
D'oh!  Trying to juggle fixes to my own stuff versus debugging `gax-google-python-v1`.